nextcord/interactions.py

```diff
@@ -50,6 +50,7 @@
     "InteractionMessage",
     "InteractionResponse",
     "PartialInteractionMessage",
+    "MessageInteraction",
```

Comment:
This should be in the `__all__` of message.py since you moved it.

Reply:
give me a couple of minutes to recoup and fix everything, getting a bit overwhelmed lol

Reply:
yes I am aware, thanks for the notice tho